inputFiles/wellbore/CasedThermoElasticWellbore_ImperfectInterfaces_base.xml

```diff
@@ -0,0 +1,583 @@
+      compressibility="5e-10"
+      thermalExpansionCoeff="1e-10"
+      viscosibility="0.0"
+      volumetricHeatCapacity="1"
+      referenceInternalEnergy="1"/>
+    <!-- SPHINX_FluidPropertiesEnd -->
+
+	<!-- SPHINX_ThermoElasticProperties -->
+    <ElasticIsotropic
+      name="casingSolid"
+      defaultDensity="7500"
+      defaultBulkModulus="159.4202899e9"
+      defaultShearModulus="86.61417323e9"
+      defaultThermalExpansionCoefficient="1.2e-5"/>
+
+    <ElasticIsotropic
+      name="cementSolid"
+      defaultDensity="2700"
+      defaultBulkModulus="2.298850575e9"
+      defaultShearModulus="1.652892562e9"
+      defaultThermalExpansionCoefficient="2.0e-5"/>
+
+    <ElasticIsotropic
+      name="rockSolid"
+      defaultDensity="2700"
+      defaultBulkModulus="5.535714286e9"
+      defaultShearModulus="3.81147541e9"
+      defaultThermalExpansionCoefficient="2.0e-5"/>
+    <!-- SPHINX_ThermoElasticPropertiesEnd -->
+    
+  </Constitutive>
+
+  <FieldSpecifications>
+   <!-- SPHINX_ContactBoundaryConditions -->
+   <FieldSpecification
+      name="frac"
+      initialCondition="1"
+      setNames="{ casingCementInterface, cementRockInterface }"
+      objectPath="faceManager"
+      fieldName="ruptureState"
+      scale="1"/>
+
+    <FieldSpecification
+      name="separableFace"
+      initialCondition="1"
+      setNames="{ casingCementInterface, cementRockInterface }"
+      objectPath="faceManager"
+      fieldName="isFaceSeparable"
+      scale="1"/>
+    <!-- SPHINX_ContactBoundaryConditionsEnd -->
+
+    <!-- SPHINX_FixedFarFieldBC -->
+    <FieldSpecification
+      name="rPosConstraint_x"
+      objectPath="nodeManager"
+      fieldName="totalDisplacement"
+      component="0"
+      scale="0.0"
+      setNames="{ rpos }"/>
+
+    <FieldSpecification
+      name="rPosConstraint_y"
+      objectPath="nodeManager"
+      fieldName="totalDisplacement"
+      component="1"
+      scale="0.0"
+      setNames="{ rpos }"/>
+    <!-- SPHINX_FixedFarFieldBCEnd -->
+
+    <!-- SPHINX_PlaneStrainAxisymmetryBC -->
+    <FieldSpecification
+      name="tNegConstraint"
+      objectPath="nodeManager"
+      fieldName="totalDisplacement"
+      component="1"
+      scale="0.0"
+      setNames="{ tneg }"/>
+
+   <FieldSpecification
+      name="tPosConstraint"
+      objectPath="nodeManager"
+      fieldName="totalDisplacement"
+      component="0"
+      scale="0.0"
+      setNames="{ tpos }"/>
+
+    <FieldSpecification
+      name="zconstraint"
+      objectPath="nodeManager"
+      fieldName="totalDisplacement"
+      component="2"
+      scale="0.0"
+      setNames="{ zneg, zpos }"/>
+    <!-- SPHINX_PlaneStrainAxisymmetryBCEnd -->
+
+    <!-- SPHINX_StressCasingInit -->
+    <FieldSpecification
+      name="initialSigma_x_casing"
+      initialCondition="1"
+      setNames="{ all }"
+      objectPath="ElementRegions/casing/casing"
+      fieldName="casingSolid_stress"
+      component="0"
+      scale="573913043.5"/>
```

Comment:
```suggestion
      scale="5.74e8"/>
```
Why such high initial stress is applied?

Reply:
Initial stress is applied to compensate with the stress generated by the initial temperature. In the casing, the stress generated by the initial temperature is high because of high elastic moduli.